pkg/insight/insightstore/insightstoretest/mock.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2021 The PipeCD Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package insightstoretest
+
+import (
+	_ "github.com/golang/mock/gomock"
```

Comment:
Can I ignore this error since it is in the same format as [pkg/filestore/filestoretest/mock.go](https://github.com/pipe-cd/pipe/blob/master/pkg/filestore/filestoretest/mock.go)?